app/javascript/packs/application.js

```diff
@@ -11,10 +11,23 @@ import restrict from '../restrict'
 import editDocumentForm from '../documents/edit'
 import MarkdownEditor from '../components/markdown-editor'
 import ContextualGuidance from '../components/contextual-guidance'
+import Raven from 'raven-js'
 
-restrict('edit-document-form', editDocumentForm)
-restrict('markdown-editor', ($el) => new MarkdownEditor($el).init())
+var $sentryDsn = document.querySelector('meta[name=sentry-dsn]')
+var $sentryCurrentEnv = document.querySelector('meta[name=sentry-current-env]')
 
-// Initialise guidance at document level
-var guidance = new ContextualGuidance()
-guidance.init(document)
+if ($sentryDsn && $sentryCurrentEnv) {
+  // set as global variable for console usage
+  window.Raven = Raven.config($sentryDsn.getAttribute('content'), {
+    environment: $sentryCurrentEnv.getAttribute('content')
+  }).install()
+}
+
+Raven.context(() => {
```

Comment:
So the docs say this is intended to wrap our 'app start' to capture additional errors, but we don't have an app here. Could we leave it out for now, to avoid confusion?